tests/support_server/setup.pm

```diff
@@ -129,6 +129,7 @@ sub setup_networks {
     $setup_script .= "systemctl restart network\n";
 
     $setup_script .= "FIXED_NIC=`grep $net_conf->{fixed}->{mac} /sys/class/net/*/address |cut -d / -f 5`\n";
+    $setup_script .= "iptables -F\n";
```

Comment:
This change looks perfectly correct, so let's hope no one relies on some default chains being available in the system ;)